src/providers/wallet/wallet.ts

```diff
@@ -1455,13 +1455,15 @@ export class WalletProvider {
           }
         })
         .catch(err => {
-          const msg =
+          let msg =
             err && err.message
               ? err.message
               : this.translate.instant(
                   'The payment was created but could not be completed. Please try again from home screen'
                 );
           this.logger.error('Sign error: ' + msg);
+          if (msg == 'Non-base58 character')
+            msg = msg + '. Please, upgrade your App to latest version.';
```

Comment:
maybe:
this.translate.instant(`${msg}. Please, upgrade your App to latest version. `)